static/js/latest-news.js

```diff
@@ -0,0 +1,63 @@
+(function() {
+  function _articleDiv(article, gtmEventLabel) {
```

Comment:
Your not using the `gtmEventLabel` param here. Do you need it? It seems to be hard coded in the template.

Reply:
I have now just implemented it, sorry.